src/applications/auth/containers/AuthApp.jsx

```diff
@@ -98,6 +98,16 @@ export class AuthApp extends React.Component {
     if (!this.state.error || hasSession()) this.validateSession();
   }
 
+  getReturnUrl = () => {
```

Comment:
@U-DON Do you have any suggestions for a different way of implementing this? Ideally we'd like to override the returnUrl that the login flow redirects back to, but the cleanest way of doing that appears to be this, which makes a site-wide assumption about `/error` in urls.

Reply:
Yeah, I'm not sure about trying to handle all redirect exceptions in this component. I haven't come up with much, but I thought about setting the `returnUrl` (maybe not in sessionStorage, but something less persistent but still univerally available; maybe a property on the `window` object?) from the component that needs a custom redirect. Then any of the auth functions that redirect and use the `RETURN_URL` value could check for the existence of the custom set `returnUrl` and use that instead of the current URL. But I haven't thought through all the ramifications.

Reply:
Let me look into this more and see if I can do a redirect from the actual error page. Let me see if there is something I can use to indicate that we came back from the auth service.